install/configure.sh

```diff
@@ -6,6 +6,31 @@
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+echo ROOT=${ROOT}
+
+if [ ! -d "${ROOT}/.clean_" ]
+then
+    echo "Initializing CWS directory... "
+    mkdir ${ROOT}/.clean_
+    # Save clean CWS before configure.sh is run and directory is modified
+    rsync -av --exclude='bpmn' ${ROOT}/* ${ROOT}/.clean_ > /dev/null 2>&1
+else
+    echo "Re-running CWS Config, backing up current configuration. Refer to .backups_ folder for all backups."
+
+    # Make backup_ folder for current CWS property version
+    BACKUP_DIR=${ROOT}/.backups_/.backup_$(date '+%F_%T')
```

Comment:
```
% echo /.backups_/.backup_$(date '+%F_%T')
/.backups_/.backup_2021-10-06_09:10:30
```
Two notes:
 - Can we go with `.backups/` and drop the unnecessary tailing underscore?
 - Let's avoid special characters (`:`) in paths.. could cause a [whole host of problems down the road](https://www.mtu.edu/umc/services/websites/writing/characters-avoid/).

I suggest we use `backup_YYYYMMDD_hhmmss` as the format here, as it will follow more conventional naming schemes. No need to hide the backup dirs either behind a dotfile, the main `.backups/` dir is already hidden.

Example:
```
% echo /.backups/.backup_$(date '+%Y%m%d_%H%M%S')
/.backups/.backup_20211006_091523
```

Reply:
Looks great! Thanks